src/api/database/bulk-api.rst

```diff
@@ -473,11 +473,15 @@ Sending multiple queries to a database
     :<header Accept: - :mimetype:`application/json`
     :<header Content-Type: :mimetype:`application/json`
     :query boolean revs: Give the revisions history
-    :<json array docs: List of document objects, with ``id``, and optionnaly
+    :<json array docs: List of document objects, with ``id``, and optionally
       ``rev`` and ``atts_since``
     :>header Content-Type: - :mimetype:`application/json`
-    :>json object results: the documents, with the additionnal ``_revisions``
-      property that lists the parent revisions if ``revs=true``
+    :>json object results: an array of results for each requested document/rev
+    pair. ``id`` key lists the requested document ID, ``docs`` contains a
```

Comment:
this line (and the ones following it) are improperly indented, breaking the build. FYI.